packages/dds/map/src/directory.ts

```diff
@@ -823,10 +832,11 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
     }
 
     /**
+     * {@inheritDoc @fluidframework/shared-object-base#SharedObjectCore.applyStashedOp}
      * @internal
      */
-    protected applyStashedOp(op) {
-        const handler = this.messageHandlers.get(op.type);
+    protected applyStashedOp(op: unknown): unknown {
+        const handler = this.messageHandlers.get((op as IDirectoryOperation).type);
         if (handler === undefined) {
```

Comment:
In this case we're implementing an interface method that explicitly takes `any`. So per the current interface contract, that would be invalid. 

It does seem like it might be worth updating SharedObjectBase to take an Op type parameter so DDS_s could be a bit more type safe, though.